R/valid.R

```diff
@@ -104,16 +104,13 @@ validate_properties <- function(object, class) {
   errors <- character()
 
   for (prop in class@properties) {
-    # Only validate static properties
-    if (!is.null(prop$getter) || !is.null(prop$setter)) {
```

Comment:
Was this basically just wrong before? i.e. if there is _only_ a setter (for whatever reason, maybe deprecation idk) then you should still be able to validate it here

Reply:
Yeah, I think so.